<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test</class>
 <widget class="QMainWindow" name="Test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>661</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>This is sample question</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="option1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>option 1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="option2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>option 2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="option3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>option 3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="option4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>option 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>250</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>submit</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Don't know</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Left</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>seconds</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
